.github/workflows/codeql-analysis.yml

```diff
@@ -46,9 +44,9 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
-    #- run: |
-    #   make bootstrap
-    #   make release
+    - run: |
+       make bootstrap
```

Comment:
this is a dotnet repo, we should probably call dotnet build at this stage instead. And maybe add a dotnet install/init step before the codeql initialization like here https://github.com/microsoftgraph/aspnetcore-webhooks-sample/blob/b04b3289f9cc24182da72406a2795eb1b1daeeee/.github/workflows/dotnetcore.yml#L16

Reply:
Thanks, it helps a lot.